### ISSUE TITLE
Enhancement request: coturn

### ISSUE THREAD

**Issue by morrisfamily** [Starbix/dockerimages#7]
really appreciate your nextcloud with docker-compose example also the plex-ldap container any chance you are working on a coturn container to go along with this for the talk app?

**Comment by Starbix** [comment]
Just to be sure, is it this project: https://github.com/coturn/coturn ?

I can look into it this week.

**Comment by morrisfamily** [comment]
That is it yes. It allows non local-network users to do voice, video, and screen sharing inside of talk on nextcloud instance.

**Comment by Starbix** [comment]
What is its advantage over the built-in one in Nextcloud?

**Comment by morrisfamily** [comment]
so my understanding was that nextcloud has a built in stun capability. However if you try to audio, video, or screen share with another user behind nat'd firewall you just get a black screen. Which has been my experience. - The built in system can handle users behind the same firewall.  - source : https://help.nextcloud.com/t/howto-setup-nextcloud-talk-with-turn-server/30794

**Comment by Starbix** [comment]
Would you help me test my coturn docker?
It's available under starbix/coturn
There's a standard config integrated, but if you want a custom config you need to mount it to /config/turnserver.conf

**Comment by morrisfamily** [comment]
I assume by standard config you mean it is the default from: https://github.com/coturn/coturn/blob/master/examples/etc/turnserver.conf

if I can try this today I already pulled the container image and deployed it. the only pieces I am unsure about is with reverse proxy setup. I have Traefik for nextcloud and if I will need labels on this container or just forward the udp ports or both.

**Comment by Starbix** [comment]
I took the config from the nextcloud link you sent.
From what I've seen you just need to forward the port (Docker and NAT), you'll probably find info about setting it up under that link

**Comment by morrisfamily** [comment]
ok thanks will let you know. If I do not get around to it today it will probably be next weekend.

**Comment by morrisfamily** [comment]
okay had some time to play with it today - and yes I just needed to forward the desired port in my case 5349 i did end up bind mounting the /config/ to make changes to the turnserver.conf i.e. my port, the paths to certs, and setting secret key and realm to my domain. 

Worked fine for 1x1 from my firewalled computer to my brothers 1200 miles away. the performance is not where I want it to be and the more I read up on turn servers the more my head hurts. not to mention when I tried to put a third person in the video chat it kind of crapped itself. I got a weird 503 error and the container quit logging and I had to force a restart. But it does work it just may be beyond my current knowledge base to trouble shoot and setup in a good way. It could have also been the talk app in nextcloud not sure. *a little frustrated*
